app/models/role.rb

```diff
@@ -0,0 +1,6 @@
+class Role < ApplicationRecord
+  has_many :person_roles
+  has_many :people, through: :person_roles
+
+  enum name: [:admin]
```

Comment:
I think the reason it is safer is that the order of the array could change over time and by clearly stating that the admin is at the zero index and always has been we avoid some of the dangers of future devs messing up our code.